openpype/hosts/houdini/plugins/create/create_vray_rop.py

```diff
@@ -54,6 +57,15 @@ def create(self, subset_name, instance_data, pre_create_data):
             "SettingsEXR_bits_per_channel": "16"   # half precision
         }
 
+        if pre_create_data.get("export_job"):
+            scene_filepath = \
+                "{export_dir}{subset_name}/{subset_name}.$F4.vrscene".format(
+                    export_dir=hou.text.expandString("$HIP/pyblish/vrscene/"),
+                    subset_name=subset_name,
+                )
+            parms["render_export_mode"] = "1"
```

Comment:
Based on [this comment](https://github.com/ynput/OpenPype/pull/5420#issuecomment-1670919365) I suppose the value should be 2 for "Export" because 1 is "Export & Render"
```suggestion
            parms["render_export_mode"] = "2"
```

> 0. No Export
> 1. Export & Render
> 2. Export

Reply:
correct yeah